schema/v0/schema.go

```diff
@@ -0,0 +1,9 @@
+// Package schema embeds the flag manifest into a code module.
+package schema
+
+import _ "embed"
+
+// Schema contains the embedded flag manifest schema.
+//
+//go:embed flag_manifest.json
+var SchemaFile string
```

Comment:
Since this is WIP, this might already be a known thing, if so, please disregard. 

I attempted to build and install this branch, but ran into SchemaPath and Schema undefined from this module after running `make generate-schema`. I made some local edits to this file to define those and get it building for now.

Is the intent for those to be defined here, or perhaps have a default schema path and an optional flag to reconfigure the schema path if needed?

Reply:
Please try again when you have a moment and make sure it's working as expected.